charts/pega/templates/pega-environment-config.yaml

```diff
@@ -59,6 +59,38 @@ data:
   CASSANDRA_TRUSTSTORE: "{{ .Values.dds.trustStore }}"
   # Path to a mounted key store file for Cassandra encryption
   CASSANDRA_KEYSTORE: "{{ .Values.dds.keyStore }}"
+  # Enable asynchronous processing of records in DDS Dataset save operation. Failures to store individual records will
+  # not interrupt Dataset save operations
+  CASSANDRA_ASYNC_PROCESSING_ENABLED: "{{ .Values.dds.asyncProcessingEnabled }}"
+  # Specify a prefix to use when creating Pega-managed keyspaces in Cassandra
```

Comment:
consistency...missing period>>Specify a prefix to use when creating Pega-managed keyspaces in Cassandra.

Reply:
Fixed